src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const employees = Array.from(document.querySelectorAll('li'));
```

Comment:
I think will be more better
```suggestion
const employees = [...document.querySelectorAll('li')];
```